exporters/metric/cortex/cortex_test.go

```diff
@@ -57,6 +77,173 @@ func TestExportKindFor(t *testing.T) {
 	}
 }
 
+func TestConvertToTimeSeries(t *testing.T) {
+	// Setup
+	exporter := Exporter{}
+
+	t.Run("handles valid checkpointSet", func(t *testing.T) {
```

Comment:
Can you help me understand why this test is separated from the table test below? At a glance the only difference I see is an `assert.Len()` call? Can we just add a `wantLen int` to the table tests to include this for each test? Please let me know if I've missed something.

Reply:
The first test is meant to test a checkpointset with a variety of different records, and the table tests are meant to test a single record type. It does make more sense to include them together and add the desired length to the tests since they're pretty much the exact same. I'll change that.